packages/@webex/plugin-meetings/test/unit/spec/stats-analyzer/index.js

```diff
@@ -1673,7 +1689,7 @@ describe('plugin-meetings', () => {
                     type: 'outbound-rtp',
                     bytesSent: 1000,
                     framesSent: 1,
-                    packetsSent: 1,
+                    packetsSent: 0,
```

Comment:
Query: why wasn't this caught before?

Reply:
@edvujic could you double check to see why this change was needed?